bundle/src/main/java/com/cognifide/cq/cqsm/core/antlr/ScriptRunner.java

```diff
@@ -168,9 +168,12 @@ public Void visitMacroExecution(MacroExecutionContext ctx) {
     public Void visitGenericCommand(GenericCommandContext ctx) {
       String commandName = ctx.IDENTIFIER().toString().toUpperCase();
       ParameterResolver parameterResolver = scriptContext.getParameterResolver();
-      List<ApmType> parameters = ctx.parameter().stream()
-          .map(parameterResolver::resolve)
-          .collect(Collectors.toList());
+      List<ApmType> parameters = Collections.emptyList();
+      if (ctx.parametersInvocation() != null) {
+        parameters = ctx.parametersInvocation().parameter().stream()
+            .map(parameterResolver::resolve)
+            .collect(Collectors.toList());

```

Comment:
You can also use [collectingAndThen](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html#collectingAndThen-java.util.stream.Collector-java.util.function.Function-) to map it to Parameters after collecting :)